content/boilerplates/kubectl-multicluster-contexts.md

```diff
@@ -1,7 +1,9 @@
 ---
 ---
-* The `kubectl` command is used to access both the `cluster1` and `cluster2` clusters with the `--context` flag.
-  Use the following command to list your contexts:
+You can use the `kubectl` command to access both the `cluster1` and `cluster2` clusters with the `--context` flag,
```

Comment:
This needs to start with a `*` or it breaks the flow of the item list in the `before you begin` sections of the owning document. I think something like this should work:
```
*   You can use the `kubectl` command to access both the `cluster1` and `cluster2` clusters with the `--context` flag,
    for example `kubectl get pods --context cluster1`.
    Use the following command to list your contexts:
```

Reply:
@frankbu Fixed in https://github.com/istio/istio.io/pull/4073/commits/5639f6765a32d51a915593ba85e9e0ab17462ccc.